package-lock.json

```diff
@@ -7373,7 +7373,135 @@
+        "@babel/helper-plugin-utils",
+        "@babel/helper-replace-supers",
+        "@babel/helper-simple-access",
+        "@babel/helper-split-export-declaration",
+        "@babel/helper-validator-identifier",
+        "@babel/helper-validator-option",
+        "@babel/helpers",
+        "@babel/highlight",
+        "@babel/parser",
+        "@babel/plugin-proposal-object-rest-spread",
+        "@babel/plugin-syntax-jsx",
+        "@babel/plugin-syntax-object-rest-spread",
+        "@babel/plugin-transform-destructuring",
+        "@babel/plugin-transform-parameters",
+        "@babel/plugin-transform-react-jsx",
+        "@babel/template",
+        "@babel/traverse",
+        "@babel/types",
+        "@types/prop-types",
+        "@types/scheduler",
+        "@types/yoga-layout",
+        "ansi-escapes",
+        "ansi-styles",
+        "ansicolors",
+        "arrify",
+        "astral-regex",
+        "auto-bind",
+        "balanced-match",
+        "brace-expansion",
+        "browserslist",
+        "caller-callsite",
+        "caller-path",
+        "callsites",
+        "caniuse-lite",
+        "cardinal",
+        "chalk",
+        "ci-info",
+        "cli-cursor",
+        "cli-truncate",
+        "color-convert",
+        "color-name",
+        "colorette",
+        "commondir",
+        "concat-map",
+        "convert-source-map",
+        "csstype",
+        "debug",
+        "electron-to-chromium",
+        "emoji-regex",
+        "escalade",
+        "escape-string-regexp",
+        "esprima",
+        "events-to-array",
+        "find-cache-dir",
+        "find-up",
+        "fs.realpath",
+        "gensync",
+        "glob",
+        "globals",
+        "has-flag",
+        "import-jsx",
+        "inflight",
+        "inherits",
+        "is-ci",
+        "is-fullwidth-code-point",
+        "js-tokens",
+        "jsesc",
+        "json5",
+        "locate-path",
+        "lodash.throttle",
+        "log-update",
+        "loose-envify",
+        "make-dir",
+        "mimic-fn",
+        "minimatch",
+        "minimist",
+        "minipass",
+        "ms",
+        "node-releases",
+        "object-assign",
+        "once",
+        "onetime",
+        "p-limit",
+        "p-locate",
+        "p-try",
+        "path-exists",
+        "path-is-absolute",
+        "pkg-dir",
+        "prop-types",
+        "punycode",
+        "react-is",
+        "react-reconciler",
+        "redeyed",
+        "resolve-from",
+        "restore-cursor",
+        "rimraf",
+        "safe-buffer",
+        "scheduler",
+        "semver",
+        "signal-exit",
+        "slice-ansi",
+        "source-map",
+        "string-length",
+        "string-width",
+        "supports-color",
+        "tap-parser",
+        "tap-yaml",
+        "to-fast-properties",
+        "type-fest",
+        "unicode-length",
+        "widest-line",
+        "wrap-ansi",
+        "wrappy",
+        "yallist",
+        "yaml",
+        "yoga-layout-prebuilt"
```

Comment:
this is completely unrelated to the change, can you just `git checkout package-lock.json` please?